### DISCUSSION TITLE
"missing required error components, refreshing..." when building in production

### DISCUSSION THREAD

**Discussion by DanielPantle** [vercel/next.js#52633] [Help]
### Summary

Hello together,

I am trying to upgrade my next.js app from version 12 to 13. I use a webhosting provider with plesk and I use TailwindCSS in my application.

When building the app in production and starting it, I get the following error when accessing the page:
`missing required error components, refreshing...`
In the console, the following network error is logged approx. every second:
`
XHRGET
https://.../
[HTTP/1.1 500 Internal Server Error 46ms]
`
The reply contains the error text from above.

As a start file, I tested `node_modules/.bin/next` and `node_modules\next\dist\bin\next`. For next.js 12, only the first one worked.

When building the app with development environment, everything works fine.

### Additional information

```js
next info:
    Operating System:
      Platform: linux
      Arch: x64
      Version: #1 SMP Debian 4.19.249-2 (2022-06-30)
    Binaries:
      Node: 16.20.1
      npm: 8.19.4
      Yarn: 1.22.19
      pnpm: N/A
    Relevant Packages:
      next: 13.4.9
      eslint-config-next: 13.4.9
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 4.9.4
    Next.js Config:
      output: N/A
```

**Comment by maxeth** [comment]
Did you figure out a solution? 

For me, this was caused by using `&` instead of `&&` in one of my scripts inside package.json. As a result, I somehow ended up running multiple next servers at the same time (?).

I used `&` intentionally to run a script in the background, but apparently this won't work.

**Reply by icyJoseph** [reply]
- https://stackoverflow.com/a/26770612

> a single & means that the preceding commands—to the immediate left of the &—should simply be run in the background.

**Reply by maxeth** [reply]
@icyJoseph 

Right. I tried to run `next dev` in the background to execute some other scripts along with `npm run dev`, but as you're probably implying, the server process doesn't stop as expected then.

**Reply by jordanlambrecht** [reply]
Lifesaver. I was going bananas until reading this and I realized I had two terminals running next dev
